plugins/modules/aci_firmware_policy.py

```diff
@@ -188,8 +243,13 @@ def main():
     argument_spec.update(aci_owner_spec())
     argument_spec.update(
         name=dict(type="str"),  # Not required for querying all objects
+        description=dict(type="str", aliases=["descr"]),
         version=dict(type="str"),
-        ignoreCompat=dict(type="bool"),
+        effective_on_reboot=dict(type="bool"),
+        ignore_compat=dict(type="bool"),
```

Comment:
should we keep alias for the old ignoreCompat attribute?

Reply:
I do not think it is much needed.